src/pocketmine/entity/Living.php

```diff
@@ -238,7 +238,7 @@ public function hasEffect(int $effectId) : bool{
 	 * @return bool
 	 */
 	public function hasEffects() : bool{
-		return count($this->effects) > 0;
+		return !empty($this->effects);
```

Comment:
checking an array with empty() is much way better than count it. In this case, we only check if empty or filled. 👍